listenbrainz_spark/ftp/download.py

```diff
@@ -61,17 +87,7 @@ def download_msid_mbid_mapping(self, directory, mapping_dump_id=None):
             Returns:
                 dest_path (str): Local path where mapping has been downloaded.
         """
-        self.connection.cwd('/pub/musicbrainz/listenbrainz/labs/mappings/msid-mbid-mapping/')
-        mappings_dump = self.list_dir()
-        req_mapping_dump = self.get_req_dump(mappings_dump, mapping_dump_id, 3)
-
-        self.connection.cwd(req_mapping_dump)
-        mapping_file_name = self.get_file_name(req_mapping_dump)
-
-        t0 = time()
-        current_app.logger.info('Downloading {} from FTP...'.format(mapping_file_name))
-        dest_path = self.download_dump(mapping_file_name, directory)
-        current_app.logger.info('Done. Total time: {:.2f} sec'.format(time() - t0))
+        dest_path = self.get_spark_dump_path(directory, mapping_dump_id, config.FTP_MSID_MBID_DIR, 3)
```

Comment:
what is the 3 at the end of this line?

Reply:
`listenbrainz-dump-61-20190708-000001-full/` : dump identifier position.